allofplos/tests/test_unittests.py

```diff
@@ -99,8 +72,8 @@ def test_class_doi1(self):
         XML file is in test directory
         """
         article = Article(class_doi, directory=TESTDATADIR)
-        self.assertEqual(article.check_if_doi_resolves(), "works", 'check_if_doi_resolves does not transform correctly for {}'.format(article.doi))
-        self.assertEqual(article.check_if_link_works(), True, 'check_if_link_works does not transform correctly for {}'.format(article.doi))
+        # self.assertEqual(article.check_if_doi_resolves(), "works", 'check_if_doi_resolves does not transform correctly for {}'.format(article.doi))
```

Comment:
Why are these commented out instead of deleted?

Reply:
Figured we might want still want these sometime for tests involving the internet?

Reply:
I'd say 
A: we can rewrite them if that time comes
and 
B: we probably can get them to work slightly differently if we switch to using pytest's semantic inference (e.g., bare asserts without saying the type of assert).